frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -81,8 +81,8 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
     componentDidUpdate(prevProps: InternalProps) {
         const { parameters, pluralForm, selectedEntity, activeTranslation } = this.props;
 
+        // We should not use parameters.entity here, because it's already set from the URL
```

Comment:
I don't think that comment is useful anymore, because in a month both of us will have forgotten the context. :P 

Also, you can remove `parameters` from the declared variables I think.